src/app/components/ContractVerificationIcon/AbiPlaygroundLink.tsx

```diff
@@ -21,6 +21,10 @@ export const AbiPlaygroundLink: FC<{
       [Layer.emerald]: `${externalLinks.dapps.abiPlayground}?network=42261&contractAddress=${address_eth}`,
       [Layer.sapphire]: `${externalLinks.dapps.abiPlayground}?network=23295&contractAddress=${address_eth}`,
     },
+    [Network.localnet]: {
+      [Layer.emerald]: '',
+      [Layer.sapphire]: '',
```

Comment:
`${externalLinks.dapps.abiPlayground}?network=23293&contractAddress=${address_eth}`

abi-playground supports it now https://github.com/oasisprotocol/abi-playground/commit/49facdc2e49ccbbf4d2074662c920ee64fa16805#diff-179ba0e03b98cb6ec52dd161e395e58906c36fba6bda405e160ea195ba9424e0L61 but localnet doesn't have sourcify verification, so it won't get activated